src/SignalRServiceExtension/Bindings/ServiceEndpointExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.SignalR;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    internal static class ServiceEndpointExtensions
```

Comment:
put it into `ServiceEndpointJsonConverter`? looks like only `ServiceEndpointJsonConverter` use it, and `ToLiteServiceEndpoint` returns a enpoint with fake endpoint is not a good idea to be internal available.